bin/update

```diff
@@ -18,8 +17,8 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
```

Comment:
Didn't we include something about having migrations recently? Does that need this line?